### PR TITLE
Update/back button

### PR DESCRIPTION
**Pull request by louise-r-blue**
This PR resolves an issue for story NZSL-32. The original story was an issue that the 'Back to Search Results' link did not render when a user added a sign to their vocab sheet. 

This update PR helps resolve a further issue as described here.

After a user clicked the 'learn more' link and then removed a sign from their vocab sheet the 'Back to search results' link did not render.

An ajax post request has been implemented that when the user selects a sign to delete, the updated vocab sheet only re-renders, making the 'Back to search results' link accessible and working for the user to navigate back to the search results.

### CODE REVIEW

File: app/assets/javascripts/search-vocab.js [hunk 1]
```diff
@@ -32,6 +43,30 @@ $(document).ready(function() {
 
       $(notice).removeClass('hide-flash').text('Sign added');
       $(vocabList).append(htmlElem);
+      removeVocabItemOnClick();
+      hideNotice();
+    }
+
+    function removeVocabItem(itemId) {
+    $.ajax({
```
Comment (Br3nda) [inline app/assets/javascripts/search-vocab.js]: indentation is messy here

File: app/views/shared/_vocab_sheet_item.html.haml [hunk 2]
```diff
@@ -1,9 +1,8 @@
-%li.vocab-sheet__item.vocab-sidebar__item
-  = image_tag "/images/signs/400-400/#{vocab_sheet_item.drawing}", class: "vocab-sidebar__image"
+%li{ class:"vocab-sheet__item vocab-sidebar__item", "data-sign-id": vocab_sheet_item.id }
```
Comment (Br3nda) [inline app/views/shared/_vocab_sheet_item.html.haml:1]: why did the class need to be moved to inside { } ?

File: app/controllers/items_controller.rb [hunk 3]
```diff
@@ -57,7 +57,7 @@ def destroy # rubocop:disable Metrics/AbcSize
     return respond_with_json_or_redirect(@item) unless request.xhr?
 
     flash[:vocab_bar_notice] = flash[:vocab_bar_error] = nil
-    render nothing: true
+    render json: nil, status: :ok
```
Comment (Br3nda) [inline app/controllers/items_controller.rb:60]: :sparkly

**Review by Br3nda — APPROVED** [review]
looks good. Couple little code linter issues.

File: app/assets/javascripts/search-vocab.js (hunk 1 above)
Comment (louise-r-blue) [reply app/assets/javascripts/search-vocab.js]: I totally missed that. thanks!

**Review by Rabid-Dan — APPROVED** [review]
good to go once that indentation has been sorted

**Pull request merged**